core/src/components/checkbox/checkbox.tsx

```diff
@@ -228,6 +287,7 @@ export class Checkbox implements ComponentInterface {
             part="label"
           >
             <slot></slot>
+            {this.renderHintText()}
```

Comment:
I noticed that the focus with VoiceOver does not highlight correctly. This is happening on the `main` branch. We should consider creating a spike to fix it.

Reply:
Hey! Are you referring to macOS or iOS?

Here's what I’m seeing:

**Chrome macOS** 

<img width="300px" src="https://github.com/user-attachments/assets/04f418e7-b733-413b-9bcf-34cc8db34111">

**Firefox macOS**

<img width="300px" src="https://github.com/user-attachments/assets/2cd6d1ee-5eb5-4a7e-857c-989e179cb487">

**iOS**

<img width="300px" src="https://github.com/user-attachments/assets/703eb92c-021b-4588-9a6b-d9b2bf9e1d97">

Could you clarify the issue you're seeing? Thanks!

Reply:
I see it on Safari macOS.

Reply:
Ah the one browser I didn't check 😂 Okay I see it, I will make a ticket!

Reply:
ROU-11658 created

Reply:
Thank you!